.github/CONTRIBUTING.md

```diff
@@ -102,14 +102,41 @@ Any new components or updates to existing components require the following:
   - Read by a screen reader.
   - Minimum contrast ratio.
 - Changes must be made for Vue 2 as well as Vue 3, `dialtone-vue2` and `dialtone-vue3` package folders respectively
+
 - Unit tests are passing locally.
-  - `nx test dialtone-vue2` or `nx test dialtone-vue3`
+  - Test all projects
+    - `nx run dialtone:test:all`
```

Comment:
doesn't it also work if you just do `nx run test` from the root directory? I thought the idea was that the commands all have the same name throughout all the packages.

Reply:
All the test commands have the same name, you can run them with `nx run-many -t test` the `nx run dialtone:test:all` is just like a shortcut `dialtone` is the name of the root package.